vizro-core/examples/_dev/app.py

```diff
@@ -1,38 +1,53 @@
 """Dev app to try things out."""
 
-import numpy as np
-import vizro.models as vm
-import vizro.plotly.express as px
-from vizro import Vizro
+import pandas as pd
+from dash import Dash, html
+from vizro.figures.undecorated import kpi_card, kpi_card_reference
 
-df = px.data.iris()
-df["species_one_long"] = np.where(
-    df["species"] == "setosa", "setosa is one common species you can select in the iris dataset.", df["species"]
-)
-df["species_long"] = df["species"] + " is one common species you can select in the iris dataset."
-df["species_very_long"] = (
-    df["species"]
-    + " is one common species you can select in the iris dataset is one common species you can select in the iris data."
+df_kpi = pd.DataFrame({"Actual": [100, 200, 700], "Reference": [100, 300, 500], "Category": ["A", "B", "C"]})
+
+# Add single CSS file figures.css or
+vizro_bootstrap = (
+    "https://raw.githubusercontent.com/mckinsey/vizro/main/vizro-core/src/vizro/static/css/vizro-bootstrap.min.css"
 )
+vizro_css = "https://raw.githubusercontent.com/mckinsey/vizro/main/vizro-core/src/vizro/static/css/figures.css"
+vizro_tokens = "https://raw.githubusercontent.com/mckinsey/vizro/main/vizro-core/src/vizro/static/css/token_names.css"
+vizro_variables = "https://raw.githubusercontent.com/mckinsey/vizro/main/vizro-core/src/vizro/static/css/variables.css"
```

Comment:
```suggestion
base = f"https://cdn.jsdelivr.net/gh/mckinsey/vizro@0.1.18/vizro-core/src/vizro/static/css/"
vizro_bootstrap = base + "vizro-bootstrap.min.css"
vizro_css = base + "figures.min.css"
vizro_tokens = base + "token_names.min.css"
vizro_variables = base + "variables.min.css"
```

The reason this didn't work is because raw github doesn't work well as a CDN - let's us jsDelivr for that (same as bootstrap uses). Note the `.min` files get created automatically, so it doesn't matter that these don't exist in our repo.

If we do things this way then we should just put all the required CSS files together into some list so you'd just do `external_stylesheets=vizro.themes` or something.

With all that said, I wonder whether we should actually do this differently and bundle the CSS files with vizro itself [like this](https://community.plotly.com/t/introducing-agridable-effortlessly-format-tabular-data-in-your-dash-app/84815/6). That would mean you don't need to add anything to your external stylesheets in order to use these components (just like when we use dmc or AgGrid components for example).

In fact I wonder whether we should just be serving all our vizro static assets that way even within a Vizro app (not just to use our components in a pure Dash app) 🤔 Let us try to understand this more and see what the pros/cons are...

Reply:
I'll pause this PR for now, let's discuss tomorrow if you're back 👍